docs/about-filecoin/ipfs-and-filecoin.md

```diff
@@ -5,38 +5,46 @@ description: Learn more about the relationship and different use-cases between I
 
-### Using IPFS
+In that sense, Filecoin can be seen as a "cold" storage layer, perfect to safely store large batches of data, while IPFS would be the "hot" storage layer, designed for the quick retrieval and distribution of content.
+
+## Which should I use?
 
-As mentioned: in IPFS, data is hosted by nodes pinning the desired data. To persist data when a user's node is offline, users must either rely on other peers to voluntarily/altruistically pin their data or use a centralized pinning service to store it.
+### Using IPFS
 
-Relying on peers to altruistically cache data might work well where one -- or multiple -- organizations share popular files on an internal network (intranet), or where strong social contracts can be used to ensure the content remains hosted and maintained long-term. Most users in the IPFS Network use a pinning service.
+- Data provided by user's own nodes. Otherwise must rely on other peers to voluntarily/altruistically storing data or on a centralized pinning service.
+- Centralized IPFS pinning services must be trusted to do their job. IPFS brings no built-in provisions to verify that data is being stored and correctly provided by the pinning service.
+- IPFS by itself works very well for popular content (with many providers), for organizations where there are incentives to sync and store data in multiple nodes and for situation where strong social contracts can be used to ensure the content remains hosted and maintained long-term.
 
 ### Using Filecoin
 
-The final option is to pin your data to a decentralized storage marketplace, such as Filecoin. In Filecoin's structure, clients make regular small payments to store data at a specified availability. Simultaneously, miners earn these payments by continuously verifying this data's integrity, storing it, and ensuring it can be retrieved quickly. This allows users to incentivize Filecoin miners to ensure their content will be live when it's needed, a clear advantage over depending solely on the generosity of other network users as required with using IPFS on its own.
+- Clients make deals with miners to store data. The network verifies that the data is indeed stored by the miners, and regular, small payments are made for the duration of the deal.
+- Miners that are unable to honor the storage conditions they offered have to pay a penalty.
+- Content retrieval might be offered by storage miners directly or by specialized retrieval miners, for a price.
+- Filecoin by itself work very well for large amounts of data to be stored for longer periods of time.
+
+### Using both
 
-## Filecoin, powered by IPFS
+Some solutions like [Powergate](../build/powergate.md) combine the best of the two worlds, backing up data on the Filecoin network and at the same time providing it through the IPFS network. This ensures availability and fast retrieval while knowing, at the same time, that the data is safely backed up by one or several Filecoin miners.
```

Comment:
```suggestion
Some solutions combine the best of the to systems, backing up data on the Filecoin network and at the same time providing the data through the IPFS network. This ensures that data is constantly available and can be retrieved quickly, while also making sure that the data is safely backed up on the Filecoin network. [Powergate](../build/powergate.md) is a service that does this.
```

Reply:
s/to/two

Reply:
It sounds weird to say powergate is a service. It's a solution, or a system, or an application, or an integration, sometimes offered as a service.

Reply:
The repo for Powergate reads:

> Powergate is a multitiered file storage API built on Filecoin and IPFS, and and index builder for Filecoin data. It's designed to be modular and extensible.

I'll just use that.